arrow-libs/core/arrow-core/src/commonTest/kotlin/arrow/core/EitherZipTest.kt

```diff
@@ -18,6 +18,25 @@ import io.kotest.property.arbitrary.short
 import io.kotest.property.arbitrary.string
 import io.kotest.property.checkAll
 
+data class Tuple10<out A, out B, out C, out D, out E, out F, out G, out H, out I, out J>(
```

Comment:
Is this still being used somewhere?

Reply:
Oh, this should have been removed

Reply:
I replaced it with `Tuple9` in the tests